### ISSUE TITLE
Eclipselink: Not able to get connection

### ISSUE THREAD

**Issue by lenzch** [OpenLiberty/open-liberty#9342]
Unwrapping a connection from the entity manager returns `null`. 
Code that is supposed to work:

```java
import javax.ejb.*;
import javax.persistence.*;

@Stateless
public class JpaTestService {
	@PersistenceContext(unitName = "myunit")	protected EntityManager em;

	@Schedule(minute = "*", hour = "*", persistent = false)
	public void doBackup(final Timer timer) {
		final java.sql.Connection conn = this.em.unwrap(java.sql.Connection.class);
		System.out.println("Connection is: " + conn);
	}
}
```
Fails with OL19.0.0.9, but used to work with OL 19.0.0.6
Might be introduced with #8252

**Comment by lenzch** [comment]
For reference, I'm using the `jpa-2.1` feature.

**Comment by dazey3** [comment]
Hello! Yes, this is a known regression delivered by #8252 and will be fixed by #9018

**Comment by lenzch** [comment]
Thanks
